### PR TITLE
issue 51: make SasFileParser API public

### PR DESCRIPTION
**Pull request by Yana-Guseva**

### CODE REVIEW

File: src/main/java/com/epam/parso/impl/SasFileParser.java [hunk 1]
```diff
@@ -1094,14 +1112,12 @@ SasFileProperties getSasFileProperties() {
         private Boolean byteOutput = false;
 
         /**
-         * The function to specify builders sasFileStream variable.
+         * The constructor that specify builders sasFileStream variable.
```
Comment (printsev) [inline src/main/java/com/epam/parso/impl/SasFileParser.java]: that specifies?

File: src/main/java/com/epam/parso/impl/SasFileParser.java [hunk 2]
```diff
@@ -508,13 +508,23 @@ private String findCompressionLiteral(String src) {
...
 
     /**
-     * The function to read next row from current sas7bdat file.
+     * The function to read and process all columns of next row from current sas7bdat file.
      *
-     * @param columnNames list of column names which should be processed.
      * @return the object array containing elements of current row.
      * @throws IOException if reading from the {@link SasFileParser#sasFileStream} stream is impossible.
      */
-    Object[] readNext(List<String> columnNames) throws IOException {
```
Comment (printsev) [inline src/main/java/com/epam/parso/impl/SasFileParser.java]: I think if we are making SasFileParser public, we need to change some JavaDocs (?) that refer to SasFileParser as "internal" class. We might want to mention this issue 51

File: src/main/java/com/epam/parso/impl/SasFileParser.java (hunk 1 above)
Comment (Yana-Guseva) [reply src/main/java/com/epam/parso/impl/SasFileParser.java]: yes, thanks

File: src/main/java/com/epam/parso/impl/SasFileParser.java (hunk 2 above)
Comment (Yana-Guseva) [reply src/main/java/com/epam/parso/impl/SasFileParser.java]: JavaDocs for SasFileParser and SasFileReaderImpl were fixed.

**Pull request closed without merging**